docs/quick_start.md

```diff
@@ -0,0 +1,195 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.5
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# napari quick start
+
++++
+
+## About napari
+
+napari is a fast, interactive, multi-dimensional image viewer, with [a vibrant plugin ecosystem](https://www.napari-hub.org/) that expands its capability to tackle various domain-specific visualization and analysis needs. It is built on Qt (for the GUI), vispy (for performant GPU-based rendering), and the scientific Python stack (numpy, scipy, and scikit-image). 
+
+napari is an open source project on [GitHub](https://github.com/napari/napari) to facilitate transparency, reuse, and extensibility. 
+
+At its core, it provides critical viewer features out-of-the-box, such as support for [large multi-dimensional data](https://napari.org/tutorials/applications/dask.html); provide [“layers”](https://napari.org/tutorials/start_index.html) to simultaneously visualize images, models, and analysis results; and easy manual, interactive annotation in 3D.
+
+This tutorial uses napari 0.4.13. <br>
+
++++
+
+## What's covered here
+
+This tutorial is for napari first-timers to give them a quick glance of what napari does, and give it a try right away. We will cover:
+
+- Installation 
+- Open napari
+- Open an image
+- Image display adjustment
+- Manually label the cell
+- Get the cell area measurement
+- Next steps
+
+Along the way, you will see how to access napari functions from [Python code](https://napari.org/api/stable/index.html) and from GUI - though for different purposes, one method might be easier than another. This quick start guide will not cover ALL possible methods but only some ways to perform basic tasks. For the more complete guide, please visit [napari.org](https://napari.org/).
+
+You will also see some examples of plugins. The core napari viewer focuses on domain-agnostic functions such as layer controls. Analyses and domain specific functions, such as reading a special file format and image segmentation, live in the realm of [plugins](https://www.napari-hub.org/).    
+
++++
+
+### Installation
+
+- For those familiar with Python:
+
+    napari can be installed on most macOS, Linux, and Windows systems with Python 3.7, 3.8, and 3.9 using pip:
+
+```python
+pip install 'napari[all]'
```

Comment:
if we're targeting newcomers, we should explain conda env creation from the installation guide.  Since installing into an existing env is one of the more common sources of bugs.  for example, `napari[all]` will install pyqt5 via pip, which will very likely break their environment if they already have pyqt installed via conda

Reply:
Yes, I will include conda env creation steps here :)

I'm also working on another PR to update the installation and getting started pages. Right now we mix Python package installation and bundled app installation together, which can be quite confusing. 
By separating these two paths, it should be clearer what prerequisites and usage cases are specific to each.